cmd/abapAddonAssemblyKitCheckCVs.go

```diff
@@ -58,7 +58,7 @@ func runAbapAddonAssemblyKitCheckCVs(config *abapAddonAssemblyKitCheckCVsOptions
 	return nil
 }
 
-//take the product part from CPE and the repositories part from the YAML file
+// take the product part from CPE and the repositories part from the YAML file
```

Comment:
You have a lot of these changes with that blank after the double slash. Is that intended ?
If not I recommend to clean up the PR to the minimal set of files you want to change.

Reply:
@radsoulbeard, that's a valid point. These changes were introduced after some formatting. Have reverted them back with [1510d73](https://github.com/SAP/jenkins-library/pull/3399/commits/1510d73e995e864f517d98b58efdb9d629af6d56)